layer.md

```diff
@@ -322,6 +310,36 @@ Note that this opaque file will apply to _all_ children, including sub-directori
 
 Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
 
+As another example, consider the following base layer:
+
+```text
+a/
+a/b/
+a/b/c/
+a/b/c/bar
+```
+
+When the next layer is created, the original `a/` directory is deleted and recreated with `a/b/c/foo`:
```

Comment:
The `a` directory itself is not deleted, only all entries in the `a` directory, if my reading is correct. To make it more clear, we could also add an `a/baz` that also gets deleted.

```suggestion
When the next layer is created, the original `a/b` directory is deleted and recreated with `a/b/c/foo`:
```